lib/helpers/analytics_helper.dart

```diff
@@ -56,12 +55,21 @@ class AnalyticsHelper {
     await _amplitude!.trackingSessionEvents(true);
   }
 
-  static Future<void> setUserId(String profileName) async {
+  static Future<void> setUserId(
+      String profileName, List<Profile> profiles) async {
     final currentUserId = await _amplitude?.getUserId();
     final isNewUser = profileName != currentUserId;
+    Map<String, double> walletTracker = {
```

Comment:
I wouldn't put this here.
This information is available (in this way) in Retool and that one is always up to date. In Amplitude its really depending on if someone is opening the app.

Reply:
A wallet tracker was part of the story's acceptance criteria. Not sure where that request came from, but it's no problem to delete